src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -64,9 +65,20 @@ public virtual ModelMetadata ContainerMetadata
         public Type ModelType => Identity.ModelType;
 
         /// <summary>
-        /// Gets the property name represented by the current instance.
+        /// Gets the name of the parameter or property if this metadata is for a parameter or property;
+        /// <see langword="null"/> otherwise i.e. if this is the metadata for a type.
         /// </summary>
-        public string PropertyName => Identity.Name;
```

Comment:
This is a change in behavior of this property's value. Is that OK?

Reply:
This _restores_ the property's pre-Preview1 behaviour.